### PR TITLE
[NOJIRA] Add documentation to 'between' functions.

### PR DESCRIPTION
**Pull request by paul-a-kennedy-rally**

### CODE REVIEW

**Review by vinit-garg — APPROVED** [review]

File: core/src/main/scala/vapors/algebra/Expr.scala [hunk 1]
```diff
@@ -28,7 +28,7 @@ import scala.collection.MapView
   * R = Return type
   * P = Captured param
   */
-sealed abstract class Expr[V, R, P] {
+sealed abstract class Expr[V, R, P] {Coursera
```
Comment (jeffmay) [inline core/src/main/scala/vapors/algebra/Expr.scala]: This looks like a bug.

**Review by jeffmay — CHANGES_REQUESTED** [review]
Looks like you have a bug in the code. Please run tests to confirm everything is working. This is still a prototype branch, so CI is not enabled here.

File: core/src/main/scala/vapors/algebra/Expr.scala (hunk 1 above)
Comment (paul-a-kennedy-rally) [reply core/src/main/scala/vapors/algebra/Expr.scala]: Yeah--was trying to clean spam out of my inbox.

**Comment by paul-a-kennedy-rally** [comment]
> Looks like you have a bug in the code. Please run tests to confirm everything is working. This is still a prototype branch, so CI is not enabled here.

Tested and passes.

**Review by jeffmay — APPROVED** [review]

**Pull request merged**